src/ui/server/controllers/insurance/business/broker/index.ts

```diff
@@ -0,0 +1,115 @@
+import { PAGES } from '../../../../content-strings';
+import { Request, Response } from '../../../../../types';
+import { TEMPLATES, ROUTES } from '../../../../constants';
+import FIELD_IDS from '../../../../constants/field-ids/insurance/exporter-business';
+// import { FIELDS } from '../../../../content-strings/fields/insurance/your-business';
+import insuranceCorePageVariables from '../../../../helpers/page-variables/core/insurance';
+import mapApplicationToFormFields from '../../../../helpers/mappings/map-application-to-form-fields';
+import generateValidationErrors from './validation';
+
+const { USING_BROKER } = FIELD_IDS.BROKER;
+
+const { BROKER } = PAGES.INSURANCE.EXPORTER_BUSINESS;
+const { BROKER: BROKER_TEMPLATE } = TEMPLATES.INSURANCE.EXPORTER_BUSINESS;
+
+export const TEMPLATE = BROKER_TEMPLATE;
+
+const { INSURANCE_ROOT, EXPORTER_BUSINESS: EXPORTER_BUSINESS_ROUTES } = ROUTES.INSURANCE;
+
+const { BROKER_ROOT, CHECK_YOUR_ANSWERS } = EXPORTER_BUSINESS_ROUTES;
+
+// const { BROKER: BROKER_FIELDS } = FIELDS;
+
+const pageVariables = (referenceNumber: number) => ({
+  FIELDS: {
+    USING_BROKER: {
+      ID: USING_BROKER,
```

Comment:
How are the field content strings coming through? Would like to have from content-strings, can see it's commented out above

Reply:
So, it will come on the other fields, this page is complicated where the page heading is the radio heading